felix/cmd/calico-bpf/commands/conntrack.go

```diff
@@ -608,26 +627,34 @@ func (cmd *conntrackFillCmd) Run(c *cobra.Command, _ []string) {
 	log.SetLevel(log.WarnLevel)
 
 	for i := 1; ; i++ {
+		var err error
 		portA := uint16(i >> 16)
 		portB := uint16(i & 0xffff)
 
-		key := conntrack.NewKey(proto, ipA, portA, ipB, portB)
-
-		if err := ctMap.Update(key[:], cmd.val); err != nil {
-			log.SetLevel(loglevel)
-			fmt.Printf("i = %+v\n", i)
-			log.Infof("Written %d entries", i-1)
-			if err == unix.E2BIG {
-				return
+		if cmd.ipv6 {
+			key := conntrack.NewKeyV6(proto, ipA, portA, ipB, portB)
+			if err = ctMap.Update(key[:], cmd.val); err == nil {
+				continue
 			}
-			log.WithError(err).Fatal("Failed to update ConntrackMap")
+		} else {
+			key := conntrack.NewKey(proto, ipA, portA, ipB, portB)
+			if err = ctMap.Update(key[:], cmd.val); err == nil {
+				continue
+			}
+		}
+
+		log.SetLevel(loglevel)
+		fmt.Printf("i = %+v\n", i)
```

Comment:
is this a debug lefover?

Reply:
This code has been here for a while. It looks like it was used for debugging purposes.
So, let's remove it ~